notebooks/MAST/m_dwarf_tess/m_dwarf_flares_tess.ipynb

```diff
@@ -0,0 +1,1285 @@
+{
```

Comment:
<p>This cleaned data looks.... ok. The first lightcurve in the list actually looks a little better to my eye; is there a reason you went with this one?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/33/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>

Reply:
<p>@ttdu this lightcurve was one of the lightcurves with more easily parseable periodograms. I could mention this in the intro, if that's helpful!</p>